pkg/innerring/internal/blockchain/blockchain.go

```diff
@@ -370,6 +370,9 @@ func New(cfg Config) (res *Blockchain, err error) {
 	cfgBaseApp.P2PNotary.UnlockWallet = cfg.Wallet
 	cfgBaseApp.RPC.StartWhenSynchronized = true
 	cfgBaseApp.RPC.MaxGasInvoke = fixedn.Fixed8FromInt64(100)
+	cfgBaseApp.RPC.SessionEnabled = true
+	// MaxIteratorResultItems must be explicitly set for rpcsrv.New because it doesn't set default
```

Comment:
I think we may safely remove this comment, https://github.com/nspcc-dev/neo-go/pull/3107 will be included into 0.102.0

Reply:
the comment explains why we set default manually (only this and no one else), think it's useful. With next release both comment and instruction will disappear